app/controllers/trending_controller.rb

```diff
@@ -12,10 +12,14 @@ def top_pages
     Page.find_by_sql('select pages.*, count(page_visits.visit_id) as total_pageviews, concat(\'https://www.gov.uk\', pages.base_path) as govuk_link from pages join page_visits on page_visits.page_id = pages.id group by pages.id limit 10')
   end
 
-  def trending_phrases
+  def most_frequent_phrases
     Phrase.find_by_sql('select count(m.phrase_id) as mentions, phrases.id, phrases.phrase_text from phrases join survey_phrases m on m.phrase_id = phrases.id join survey_answers sa on sa.id = m.survey_answer_id join surveys s on s.id = sa.survey_id group by (m.phrase_id, phrases.id, phrases.phrase_text) order by mentions desc limit 10')
   end
 
+  def top_user_groups
+    UserGroup.top_user_groups_by_date(Date.new(2020, 4, 1), Date.new(2020, 4, 7))
```

Comment:
More of a thought than a requirement, I think I'm right in guessing these are the dates for which the data is valid - if we're going to have the same dates in multiple places it would be good to extract it out somehow. Not an immediate priority though